src/ListItem/ListItem.react.js

```diff
@@ -236,6 +241,13 @@ class ListItem extends PureComponent {
             onPress(onPressValue);
         }
     };
+    onListItemLongPressed = () => {
```

Comment:
Passing the SyntheticEvent with press events, mainly for onPressOut, is helpful when needing to look at the event's targets. Might be somewhat uncommon, but helpful for things like skipping the press event when the held touch is released outside the touchable boundary (a bit tricky with nesting children, the event target data is needed to handle that too).